app/views/simulator/edit.html.erb

```diff
@@ -329,4 +332,33 @@
       <script src="/js/combinationalAnalysis.js"></script>
       <script src="/js/Quin_McCluskey.js"></script>
     <% end %>
-    <script src="/js/canvas2svg.js"></script>
\ No newline at end of file
+    <script src="/js/canvas2svg.js"></script>
+    
+    <script type="text/javascript">
```

Comment:
It would be better if you put this code into a separate js file or put it inside the logix.js .if you have some free time.
but it will be refactored any way later